Libraries/oneVPL/dpcpp-blur/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@
 #
 # SPDX-License-Identifier: MIT
 # ##############################################################################
-cmake_minimum_required(VERSION 3.10.2)
+cmake_minimum_required(VERSION 3.4)
```

Comment:
This change seems dangerous, as you are reducing the minimum required version rather than increasing it.  Please have this reviewed by the sample owner.

Comment applies to all the oneVPL designs.

Reply:
Mistake fixing